### ISSUE TITLE
Bug: ProtonVPN - South African Server Updates

### ISSUE THREAD

**Issue by dragonhelios** [qdm12/gluetun#707]
### Is this urgent?

No

### Host OS

Unraid

### CPU arch

x86_64

### VPN service provider

ProtonVPN

### What are you using to run the container

Portainer

### What is the version of Gluetun

Running version latest built on 2021-11-02 (commit 678e23c)

### What's the problem 🤔

Proton has decomissioned a few of their older South African servers and brought quite a few new ones online. when specifying a new server using SERVER_HOSTNAME using either the new hostname (eg node-za-01.protonvpn.net) or IP address it does not match the list of accepted servers.

Specifying just the country also does not work since the server it tries to connect to za-05.protonvpn.com which has been decommisioned (according to proton support).

Dowloading the individual server files and connecting using an OpenVPN client resulted in the following:
1. country based config uses za.protonvpn.com. Connecting to this server results in a connection timeout for me. I've got an open issue with Proton regarding this.
2. using server based configs connects using an OpenVPN client:
    a. non plus servers (ZA#13 to ZA#15): config files all use za-13.protonvpn.net
    b. plus servers (ZA#16 to ZA#24): config files all use node-za-01.protonvpn.net

[_vpn_proton_logs (country only specified).txt](https://github.com/qdm12/gluetun/files/7482419/_vpn_proton_logs.country.only.specified.txt)
[_vpn_proton_logs (server_hostname specified).txt](https://github.com/qdm12/gluetun/files/7482420/_vpn_proton_logs.server_hostname.specified.txt)

### Share your logs

```log
attached log file dumps
```

### Share your configuration

```yml
services:
  vpn_proton:
    image: qmcgaw/gluetun:latest
    container_name: vpn_proton
    hostname: vpn_proton
    cap_add:
      - NET_ADMIN
    network_mode: bridge
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock:ro
      - /mnt/app_data/appdata/docker_data/gluetun:/gluetun
    environment:
      - OPENVPN_USER=usename
      - OPENVPN_PASSWORD=password
      - VPNSP=protonvpn
      - COUNTRY=South Africa
      - TZ=Africa/Johannesburg
      - FIREWALL_OUTBOUND_SUBNETS=192.168.1.0/24
      - PROTOCOL=tcp
      - DOT=off
      - DNS_PLAINTEXT_ADDRESS=10.7.7.1
      - SERVER_HOSTNAME=za-13.protonvpn.net
    healthcheck:
      test: ping -c 1 www.google.com || exit 1
      interval: 30s
      timeout: 10s
      retries: 3      
    restart: always
```

**Comment by dragonhelios** [comment]
To add to above: Proton support just resolved the problem using country specific configuration ie: za.protonvpn.com

**Comment by qdm12** [comment]
Check out https://github.com/qdm12/gluetun/wiki/Updating-servers

I'll update the servers built-in the program in the coming days.

**Comment by dragonhelios** [comment]
Thanks.

**Comment by qdm12** [comment]
This should be fixed in 142238e8b70833e95fa9a5e8a091fd748fde3efe in the latest image.
